app/inputs/select_with_modal_help_input.rb

```diff
@@ -31,6 +31,18 @@ def build_field(value, _index)
       @rendered_first_element = true
 
       html_options.merge!(options.slice(:include_blank))
+      html_options = add_force_option_data_to_html_options(value, html_options)
       template.select_tag(attribute_name, template.options_for_select(select_options, value), html_options)
     end
+
+    def add_force_option_data_to_html_options(value, html_options)
+      if value.present?
+        unless RightsService.active?(value)
+          html_options[:class] << "force-select"
```

Comment:
Does `'force-select'` need a space in the string so that you don't get, e.g., `<span class="foo bar bazforce-select">` ?